sdk/device_manager.go

```diff
@@ -96,6 +97,7 @@ func newDeviceManager(plugin *Plugin) *deviceManager {
 		aliasCache:     NewAliasCache(),
 		devices:        make(map[string]*Device),
 		handlers:       make(map[string]*DeviceHandler),
+		p:              plugin,
```

Comment:
Could we please spell out p? (plugin or something)

Reply:
👍 
should be updated in the latest commit